pegasus/sites.v3/hourofcode.com/i18n/en.yml

```
@@ -2552,7 +2552,7 @@
   tutorial_codehsdata_longdescription: "Learn about the power of using data visualizations to display data in meaningful and easy to understand ways. Use our graph generator to create visually appealing graphs and learn how coding is used to create beautiful data."
   tutorial_educodecake_longdescription: "Learn HTML as you bake a cake with chef Roland. Shape and style your custom creation with your newfound knowledge of tags and classes."
   tutorial_ttsecond_longdescription: "Can you deliver the Open Protocol? You've snuck aboard the capital ship, and now you must evade the security drones long enough to open access for all. Quickly examine conditional statements to find out whether you're safe or not from the dastardly cybervision. If not, it's to the airlock!"
-  tutorial_bsdtrivia_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Trivia Game Maker focuses on building an interactive trivia game where you also get to come up with the trivia questions."
+  tutorial_bsdtrivia_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Trivia Game Maker focuses on building an interactive trivia game where you also get to come up with the trivia questions. #codeisyourvoice"
```

Adding missing spaces
```suggestion
  tutorial_bsdtrivia_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Trivia Game Maker focuses on building an interactive trivia game where you also get to come up with the trivia questions. #codeisyourvoice"
```